specifyweb/frontend/js_src/lib/components/FormCells/index.tsx

```diff
@@ -65,10 +66,16 @@ const cellRenderers: {
       />
     );
   },
-  Label({ cellData: { text, labelForCellId, title }, formatId, align }) {
+  Label({
+    cellData: { text, labelForCellId, title },
+    formatId,
+    align,
+    isBold,
+  }) {
     const style: React.CSSProperties = {
       textAlign:
         align === 'right' ? 'right' : align === 'center' ? 'center' : undefined,
+      fontSize: isBold ? 'bold' : undefined,
```

Comment:
`fontSize` is not correct here. did you mean `fontWeight`?
and are you trying to set that style here if you are already doing that in `DataEntry.Cell`?